sts-e2e-tests/src/test/java/de/adorsys/sts/tests/e2e/AdminControllerJpaTest.java

```diff
@@ -0,0 +1,82 @@
+package de.adorsys.sts.tests.e2e;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import de.adorsys.sts.persistence.jpa.config.EnableJpaPersistence;
+import de.adorsys.sts.resourceserver.model.ResourceServer;
+import de.adorsys.sts.tests.BaseEndpointTest;
+import de.adorsys.sts.tests.config.WithAdminConfig;
+import de.adorsys.sts.tests.config.WithoutWebSecurityConfig;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@EnableJpaPersistence
+@ContextConfiguration(classes = {WithAdminConfig.class, WithoutWebSecurityConfig.class})
+class AdminControllerJpaTest extends BaseEndpointTest {
+
+    @Test
+    @SneakyThrows
+    void getResourseServersTest() {
+        mvc.perform(get("/admin/resourceServer/")
```

Comment:
It is better to read json fixture from file, or to perform assertions using json path